python/cugraph/tests/dask/opg_pagerank_testing.py

```diff
@@ -38,10 +35,28 @@ def test_dask_pagerank():
                              names=['src', 'dst', 'value'],
                              dtype=['int32', 'int32', 'float32'])
 
-    g = cugraph.DiGraph()
-    g.from_dask_cudf_edgelist(ddf)
+    df = cudf.read_csv(input_data_path,
+                       delimiter=' ',
+                       names=['src', 'dst', 'value'],
+                       dtype=['int32', 'int32', 'float32'])
 
-    dcg.pagerank(g)
+    g = cugraph.DiGraph()
+    g.from_cudf_edgelist(df, 'src', 'dst')
+
+    dg = cugraph.DiGraph()
+    dg.from_dask_cudf_edgelist(ddf)
+
+    expected_pr = cugraph.pagerank(g)
+    result_pr = dcg.pagerank(dg)
```

Comment:
`dcg` -> `dask_cugraph`

Reply:
we import cugraph.dask as dcg and run pagerank. This is to follow cuml's import structure--> from cuml.dask import TheModel

Reply:
dask cuDF features are imported from `dask_cudf` though.

Reply:
To be discussed in the near future.